### DISCUSSION TITLE
Add a way to edit the current health management system

### DISCUSSION THREAD

**Discussion by A6sT** [Quaver/Quaver#4796] [Feature Request]
### Is your feature request related to a problem? Please describe.

I've always had issue with the way Quaver handles the health system in game. For example, while playing maps that are a bit above my ability to play, I would like to be allowed to miss a bit more (especially on the LN maps), so that I could still actually play, while having the feeling that I understand what's happening in the map at the same time (kind of having the feeling that i'm improving, even though i haven't passed the map in the end).

The issue as of today is that, as you may have understood it, Quaver's health managment system is way to punitive to use that playstyle. I'm a 4K player, so I might be wrong, but it's even worse when we take a look at 7K, where, for example, missing a single chord (7 notes on the same line) with small LN could almost instantly kill you. Let's also not forget that, with futures updates will come support for 1 -> 10K. If the current way health is handled doesn't change, it might actually be hard to even play maps over 7K (keep in mind that i'm only a 4k player, i don't really know a lot about higher key count gameplay)

However, I do understand why the game decided to go in that direction when it comes to health managment, I just feel like that it'd be awesome to have a way to edit the way it's currently being handled. This would allow for a lot of players of any range of level to actually play the game as they want.

### Describe the solution you'd like

A really cool feature that we have already is the ability to create custom judgment windows. This feature could be enhanced to also have a way to manage the way the health is being consumed for each judgment made. 
For example, I could make a judgment windows where Great judgment would give me a bit more life than the standard judgment windows currently gives, which would make me be able to clear maps that I was barely dying on in the past.
This could even create some new way of playing the game, where you would have your own judgment windows where Marvelous give you no health, while every other judgments makes you loose health. With this, every maps nows becomes a battle where you have to be as precise as possible to survive.

The main thing to be carefull about with the addition of this feature is the way score are handled with custom health. Like the way it's currently working with custom timing windows, health will also have to be converted in the score submission process. If it wasn't understood by now, I'll clarify it just in case, but scores made with custom health settings shouldn't be submitted if the player end up dying once the score is converted to the standard judgment window.

I might have missed some key points to this, so if anyone is inspired by this idea, feel free to add your own ideas under this.

**Comment by OWazabi** [comment]
I most def agree, since I started VSRGs my health has gotten worse and worse.
